addons/website_sale/models/res_partner.py

```diff
@@ -19,7 +19,6 @@ def _compute_last_website_so_id(self):
             if website and not is_public:
                 partner.last_website_so_id = SaleOrder.search([
                     ('partner_id', '=', partner.id),
-                    ('team_id.team_type', '=', 'website'),
```

Comment:
About that, do you have an easy/practical way to [keep commit order](https://help.github.com/articles/why-are-my-commits-in-the-wrong-order/) on github.com after an interactive rebase?
Right now, I need to `git rebase -i HEAD~9` and replace `pick` by `r` on every commit to rewrite their message to have a correct date order and so a correct github.com order, but it is kinda annoying..

Reply:
Actually found something, in case you need it:
```bash
[alias]
    reorder = "!f() { git rebase \"$1\" --ignore-date -x 'git commit --amend -C HEAD --date=\"$(date -R)\" && sleep 0.10'; }; f"
```
Then just `git reorder HEAD~8`

Reply:
Done at https://github.com/odoo/odoo/pull/31669